server/files/migrations/0003_more_updates.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1 on 2018-09-27 19:23
```

Comment:
We should squash all these migrations before merging. Basically delete all the migrations then generate a new one via:

    ./manage.py makemigrations -n add_file_model

lmk if you want me to help with this.